### PR TITLE
Correct mistakes in Japanese locale

### PR DESCRIPTION
**Pull request by kosuke316**
## Pull Request Checklist

Thank you for taking the time to improve Arrow! Before submitting your pull request, please check all *appropriate* boxes:

- [ ] 🧪  Added **tests** for changed code.
- [x] 🛠️  All tests **pass** when run locally (run `tox` or `make test` to find out!).
- [x] 🧹  All linting checks **pass** when run locally (run `tox -e lint` or `make lint` to find out!).
- [x] 📚  Updated **documentation** for changed code.
- [x] ⏩  Code is **up-to-date** with the `master` branch.

If you have *any* questions about your code changes or any of the points above, please submit your questions along with the pull request and we will try our best to help!

### CODE REVIEW

**Comment by kosuke316** [comment]
@krisfremen Thanks for the reply! I've also added a test feature. I hope you can check it out.

File: contributors/kosuke316.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Correction of japanese mistakes.
```
Comment (systemcatch) [inline contributors/kosuke316.md:1]: Hey @kosuke316 thanks for the PR, we don't maintain this style of contributor log so need this to be removed.

**Review by systemcatch — CHANGES_REQUESTED** [review]
All good bar one comment.

File: contributors/kosuke316.md (hunk 1 above)
Comment (kosuke316) [reply contributors/kosuke316.md]: sorry, I got it!

**Review by systemcatch — APPROVED** [review]

**Pull request merged**